sdk/communication/communication-email/swagger/README.md

```diff
@@ -0,0 +1,25 @@
+# Azure Communication Services Email REST API Client
+
+> see [https://aka.ms/autorest](https://aka.ms/autorest)
+
+## Configuration
+
+```yaml
+package-name: "@azure/communication-email"
+title: EmailRestApiClient
+description: Email REST API Client
+generate-metadata: false
+license-header: MICROSOFT_MIT_NO_VERSION
+output-folder: ../src/generated
+tag: package-2021-10-01-preview
+package-version: 1.0.0-beta.1
+require: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/specification/communication/data-plane/Email/readme.md
+model-date-time-as-string: false
+optional-response-headers: true
+typescript: true
+azure-arm: false
+add-credentials: false
+v3: true
+use-extension:
+  "@autorest/typescript": "6.0.0-dev.20200623.2"
```

Comment:
This is a fairly old version of  `@autorest/typescript` would you mind generating with "6.0.0-beta.19"

Reply:
Filling a repair item to make this change.